src/common/logger/child-logger.injectable.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import type { Logger } from "../logger";
+import loggerInjectable from "../logger.injectable";
+
+const childLoggerInjectable = getInjectable({
+  id: "child-logger",
+  instantiate: (di, prefix): Logger => {
+    const logger = di.inject(loggerInjectable);
+
+    return {
```

Comment:
Note: it's a bad practise here and elsewhere to bundle all these responsibilities together.

Reply:
Discussion handled IRL.